packages/perseus/src/widgets/interactive-graphs/graphs/circle.tsx

```diff
@@ -221,3 +207,59 @@ function crossProduct<A, B>(as: A[], bs: B[]): [A, B][] {
     }
     return result;
 }
+
+function CircleGraphDescription({state}: {state: CircleGraphState}) {
```

Comment:
this is the same as `state: CircleGraphState` right? Or is there an additional benefit to the destructing in this case?

Reply:
I think that works too. I _think_ the reason it was done this way (I was following what Ben did for the point description) is so it follows the prop structure the way they're standardly written for other functional components.